src/hooks/useAiAssistant.ts

```diff
@@ -77,6 +77,7 @@ function useAiHook(integrationId: string, aiQuery: boolean, profileId?: string):
         return squid.ai().assistant(integrationId).profile(profileId).chat(question, aiAssistantOptions);
       }
     },
+    {},
     '',
```

Comment:
maybe the initial value needs to move to the options?

Reply:
Done.